build/config.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
Interesting that there are a few different places config comes from now:

- `build/config.json`
- `.env`
- `test.env`

I don't have to solve this now, but I'd definitely like to figure out clean dividing lines here. Maybe easier if/when I get rid of the templates steps.